country/finland.md

```diff
@@ -1,28 +1,22 @@
 # Finland
-
-Finland, being part of the European Union, follows the GDPR that governs email marketing and data processing in General. They also have Act on Electronic Communication Services, which implements the E-Privacy Directive, this works together with the GDPR to regulate email marketing.
+Finland, being part of the European Union, follows the GDPR that governs email marketing and data processing. They also have the Act on Electronic Communication Services, which implements the E-Privacy Directive. Both are used to regulate email marketing.
 
 ## Content required
-
 All emails sent for marketing purposes should have a clear identification of the sender name, mailing address and a clear identification of the sender. The law requires you to have these as compulsory data to be mentioned in your marketing emails.
 
 ## Consent
-
 You can send marketing emails to those who have provided explicit consent to receive marketing emails from you.
 
 Email is also allowed if the recipient has not specifically opted out and if you obtain the customer's contact details in the context of sale of a product or service. You cannot sell the data, however you can use the contact information for direct marketing of your own similar products. This is essentially soft opt-in using implied consent.
 
 ## Notable exceptions or requirements
-
 As Finland follows the GDPR, it requires that your company must have a data protection officer. The person is in charge of maintaining and enforcing data security standards.
 
 ## Penalties
-
-The maximum fine available under the GDPR is up to €20 million, or 4% annual global turnover – whichever is higher. "The Office of the Data Protection Ombudsman" will be the body that will safeguard rights and freedoms when processing personal data.
+The maximum fine available under the GDPR is up to 20 million EUR, or 4% annual global turnover – whichever is higher. "The Office of the Data Protection Ombudsman" is the body that will safeguard rights and freedoms when processing personal data.
```

Comment:
> "The Office of the Data Protection Ombudsman"
We're a bit inconsistent in where we use quotation marks when referring to the authority. They were removed in some (i.e Denmark) but not here.

Reply:
Good point. I think I opted for removing quotes as the whole guide is littered with references, the majority of which aren't quoted. I missed this one and a few others. Corrected in https://github.com/threeheartsdigital/email-marketing-regulations/pull/29/commits/2164ae6b39611e3c77293175add2c8dfc3cdbb3f